smart-contracts/contracts/cl-vault/src/msg.rs

```diff
@@ -137,4 +137,10 @@ pub struct InstantiateMsg {
 }
 
 #[cw_serde]
-pub struct MigrateMsg {}
+pub struct MigrateMsg {
+    /// The ID of the new pool
+    pub pool_id: u64,
+    /// position details
+    pub lower_tick: i64,
```

Comment:
nit: the migration would be more minimal and less error prone if the `lower_tick` and `upper_tick` were copied from the old position

Reply:
[7b3d713](https://github.com/quasar-finance/quasar/pull/555/commits/7b3d71311efd521afa75bf8fa6b92d47c6d89a8a)